inst/modeldb/PK_2cmt_tvcl_des.R

```diff
@@ -0,0 +1,34 @@
+PK_2cmt_tvcl_des <- function() {
+  description <- "Two compartment PK model with time-dependent clearance using differential equations"
+  ini({
+    lclts <- -1.6 ; label("Time-stationary clearance (CLTS)")
+    ltmax <- -1.5 ; label("Typical value of the maximal change of clearance relative to baseline (Tmax)")
+    lgamma <- 0 ; label("Hill coefficient")
```

Comment:
Please clarify the label to something like `label("Hill coefficient for time-variant clearance")`.  That will allow the model to more clearly integrate with future models that may have a hill coefficient for a pharmacodynamic effect.